.cypress/cypress/integration/bathnes.js

```diff
@@ -39,3 +39,12 @@ it('uses the Curo Group housing layer correctly', function() {
     cy.pickCategory('Dog fouling');
     cy.contains('Maintained by Curo Group').should('be.visible');
 });
+
+it('handles code names with spaces without error', function() {
+    cy.visit('http://bathnes.localhost:3001/report/new?longitude=-2.3659276latitude=51.379009');
```

Comment:
There's a missing `&` between the two parameters in the URL. I think if you add that the tests should pass 🤞 

```suggestion
    cy.visit('http://bathnes.localhost:3001/report/new?longitude=-2.3659276&latitude=51.379009');
```